tasks/main.yml

```diff
@@ -14,74 +14,31 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 #
-- name: Check for 'insights-client' RPM availability
-  yum:
-    list: insights-client
-  register: yum_list_task
-  become: true
-
-- name: Install 'insights-client' if it is available
+- name: Install 'insights-client'
   yum:
     name: insights-client
-  when: yum_list_task.results|length > 0
   become: true
 
 - name: Set Insights Config Name to 'insights-client'
   set_fact:
     insights_name: 'insights-client'
```

Comment:
I think you just want a var in a defaults.yml role file for this, `set_fact` seems not appropriate

Reply:
Oops.  The above comment was intended for the `main.yml` usage of the `display_name` parameter

Reply:
Yeah that step is going away completely. Left over from when insights_name could be something other than 'insights-client'

Reply:
Awesome, thanks! 👍